soc/arm/arm/musca_a/Kconfig.soc

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+prompt "ARM Musca SoCs"
+depends on SOC_SERIES_MUSCA
+
+config SOC_V2M_MUSCA_A
+	bool "ARM Cortex-M33 SMM-SSE-200 on V2M-MUSCA"
+	select CPU_CORTEX_M33
+	select ARMV8_M_SE
```

Comment:
You should not directly select ARMV8M_SE.
Instead, you need to select CPU_HAS_ARM_SAU, to indicate the TEE capabilities of the SOC.

Reply:
ditto